packages/visx-demo/tsconfig.json

```diff
@@ -14,11 +14,12 @@
     "moduleResolution": "node",
     "composite": true,
     "outDir": "lib",
-    "resolveJsonModule": false,
+    "resolveJsonModule": true,
```

Comment:
ah, I think there's an issue where next forces this when running the demo app locally, but I think it fails the TS generation in CI. let's see if it passes with this change, else I'd have you revert it for merging (and separately need to fix the issue)

Reply:
I reverted this change in https://github.com/airbnb/visx/pull/1767/commits/f7dddefabbf3bca56cfe67191b691c3452776390. Yeah the file was updated by Next when I ran these commands:

```sh
cd packages/visx-demo
NODE_OPTIONS=--openssl-legacy-provider yarn dev
```

(I had to use `NODE_OPTIONS` to avoid `Error: error:0308010C:digital envelope routines::unsupported` in Node 18.18.2)

It would be nice to update `packages/visx-demo/tsconfig.json` but I guess we can do this in a separate DX-focused PR.